workers/unity/Assets/Generated/Source/improbable/gdk/tests/ExhaustiveBlittableSingularTranslation.cs

```diff
@@ -55,6 +48,35 @@ public override void OnAddComponent(AddComponentOp op)
                 entityManager.AddComponentData(entity, data);
                 entityManager.AddComponentData(entity, new NotAuthoritative<SpatialOSExhaustiveBlittableSingular>());
 
+                var updatesComponent = new SpatialOSExhaustiveBlittableSingular.ReceivedUpdates
+                {
+                    handle = ReferenceTypeProviders.UpdatesProvider.Allocate(World)
+                };
+                
+                ReferenceTypeProviders.UpdatesProvider.Set(updatesComponent.handle, new List<SpatialOSExhaustiveBlittableSingular.Update>());
+                entityManager.AddComponentData(entity, updatesComponent);
+
+                var update = new SpatialOSExhaustiveBlittableSingular.Update 
+                {
+                    Field1 = data.Field1,
+                    Field2 = data.Field2,
+                    Field4 = data.Field4,
```

Comment:
why is field 3 skipped?

Reply:
Because someone couldn't count and skipped it in the schema :trollface:

Reply:
As an aside we should change these names anyway and make the field descriptive of what it is. Otherwise this is hard to distinguish if other codegen that depends on this is ignoring field names or not.